en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -710,7 +710,7 @@ following in your .htaccess to ensure this behaviour is activated.
 </IfModule>
 ```
 
-As of Silverstripe CMS 4, this behaviour is on by default, and the environment variable is no longer required. For correct operation, it is necessary to always set `SS_TRUSTED_PROXY_IPS` if using a proxy.
+This behaviour is on by default; the environment variable is not required. For correct operation, it is necessary to always set `SS_TRUSTED_PROXY_IPS` if using a proxy.
```

Comment:
Is this correct to have a semicolon: 
`... behaviour is on by default ;  ...`

Reply:
Yup.
https://www.merriam-webster.com/words-at-play/a-guide-to-using-semicolons
> A semicolon separates related independent clauses that are joined without a coordinating conjunction, such as _and_:
> > Mabel the Cat had made a big pot of goulash; it was simmering on the stove.